File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 4/14/2026, 12:48:43 PM
class Solution:
    def containsDuplicate(self, nums: List [int]) -> bool:
        s_nums = set(nums)

        if len(s_nums) == len(nums):
            return False
        else:
            return True



        

File: 217-ContainsDuplicate/217-ContainsDuplicate.js
// Last updated: 4/14/2026, 12:48:38 PM
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let num = new Set(nums)
    let N = num.size
    let n = nums.length
    if(N !== n){
        return true
    }else{
        return false
    }
};